### ISSUE TITLE
package link might need fixing

### ISSUE THREAD

**Issue by xinyioffice** [quanser/Quanser_Interactive_Labs_Resources#1]
Dear Quanser Tech Team,

I tried to run these tutorials with the up-to-date Quanser Python packages (2023.4.13), but there is an `ImportError`:

```bash
Traceback (most recent call last):
  File "e:\GitHub_Repos\QLabs\Quanser_Interactive_Labs_Resources\python\tutorials\qcar_tutorial1.py", line 14, in <module>
    from qvl.qlabs import QuanserInteractiveLabs
  File "C:\Users\lenovo\Documents\Quanser\libraries\python\qvl\qlabs.py", line 1, in <module>
    from quanser.communications import Stream, StreamError, PollFlag, Timeout
ImportError: cannot import name 'Timeout' from 'quanser.communications' (E:\miniconda\envs\Quanser\Lib\site-packages\quanser\communications\__init__.py)
```

I have checked the [docs](https://docs.quanser.com/quarc/documentation/python/common/Types/types.html#quanser.common.types.Timeout).

It seems that you have changed the `Timeout` type from `Quanser Communications API` to `Quanser Common API` in the most recent updates.

But the `qlabs.py` from the QVL still import `Timeout` from `quanser.communications`.

I tried to manually change the `qlabs.py` from this

```python
from quanser.communications import Stream, StreamError, PollFlag, Timeout
```

to this

```python
from quanser.communications import Stream, StreamError, PollFlag
from quanser.common import Timeout
```

Then the error is fixed.

I don't exactly know how is this QVL generated, so can you please fix this?

**Comment by xinyioffice** [comment]
Sorry, I check the repo PR again, the commit 1acd879 and 9b657e0 have fixed this.

The reason why this would happen to me is that when updated the Quanser Python packages, I reinstalled the QVL, but due to network issue, it couldn't clone the repo with `pygit`:

```
Installing Quanser virtual libraries...
Error: Failed to install files from GitHub (Note: this requires a valid internet connection).
Would you like to skip this part for now and finish with the rest of the installation? (y,[n])
```

I chose to skip this part, so the APIs are new, but the QVL remains the old version.

I used a proxy to clone the repo properly, then solved this issue.

Sorry for opening an issue, as I thought it was a bug. But actually, it was my own operational problem.
